packages/features/ee/teams/components/AddNewTeamMembers.tsx

```diff
@@ -33,11 +34,21 @@ type FormValues = {
 const AddNewTeamMembers = () => {
   const searchParams = useCompatSearchParams();
   const session = useSession();
+  const telemetry = useTelemetry();
+
   const teamId = searchParams?.get("id") ? Number(searchParams.get("id")) : -1;
   const teamQuery = trpc.viewer.teams.get.useQuery(
     { teamId },
     { enabled: session.status === "authenticated" }
   );
+
+  useEffect(() => {
+    const { event } = router.query;
+    if (event === "team_created") {
+      telemetry.event(telemetryEventTypes.team_created);
+    }
+  }, []);
```

Comment:
When coming to this page after the checkout session, we should trigger the created team event.